src/css/citizen-engagement-app.webflow.css

```diff
@@ -1988,6 +1988,11 @@ strong {
   flex: 0 0 auto;
 }
 
+.link-text.form-submit {
+  display: inline-block;
+  margin-top: 8px;
+}
+
```

Comment:
This is indeed not ok - this will get overwritten with the next import.

your margin should be fixed if you make sure that the page modal content is wrapped in a 
`.grid--default`
 https://citizen-engagement-app.webflow.io/documentation#grids but you might need to wrap the button in a div for now to make it `display: block` - please link to this comment and ask matt in slack what you need to do to put the submit button on its own row. How is this done on other forms? what's different here?

Reply:
Thanks for the feedback.

I only had to change the display to `inline-block` to allow the `margin-top` to work. 

So it's a non-issue and we can exclude these changes entirely, it just means there's no margin which I think is fine until Matt can add the styling on his side.

Reply:
With no CSS changes:

![forgot-password-no-margin](https://user-images.githubusercontent.com/7869992/106144804-63512280-617d-11eb-8d71-82ae57f6a4a3.png)